src/pages/_app.tsx

```diff
@@ -84,11 +84,9 @@ function MyApp({
           ) : (
             <BreadcrumbsAppenderAG />
           )}
-          <ContentWrapperStyled>
+          <ContentWrapperStyled tabIndex={-1} id="maincontent">
```

Comment:
Hum, har det noen forskjell å bruke denne vs den som ligger den? Jeg la inn den main-greia med role=main siden axe devtools anbefalte det

Reply:
Main-taggen har native rollen `main` så da er det anbefalt å bruke den siden den har det innebygd. Fjernet den andre for å få det litt mer ryddig, mindre elementer. `id="maincontent"` lar dekoratøren skippe til main-content.